test/geometry/transform/test_pyramid.py

```diff
@@ -192,3 +192,31 @@ def test_gradcheck(self, device, dtype):
             raise_exception=True,
             fast_mode=True,
         )
+
+
+class TestUpscaleDouble:
```

Comment:
```suggestion
class TestUpscaleDouble(BaseTester):
```
`from kornia.testing import BaseTester`